### PR TITLE
Resolved conflicts : Scroll To Top feature

### PR DESCRIPTION
**Pull request by Deepanshu0703**
Making this new branch and PR so that old conflicts will be resolved.

### Screenshots

Screenshot (5)

### CODE REVIEW

**Comment by Deepanshu0703** [comment]
I have made the PR #114 again plz ignore that PR of adding scroll to Top option

File: components/GoToTop.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    position: fixed;
+    bottom: 1rem;
+    right: 2rem;
+    z-index: 999;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+
+    &--icon {
```
Comment (AmanJain18) [inline components/GoToTop.js:62]: remove thse animation

File: components/GoToTop.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+const GoToTop = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  const goToBtn = () => {
+    window.scrollTo({ top: 0, left: 0, behavior: "smooth" });
+  };
+
+  const listenToScroll = () => {
+    let heightToHidden = 20;
```
[the remark this answers is not in the document]
Comment (Deepanshu0703) [reply components/GoToTop.js]: Increased it to 200

File: components/GoToTop.js (hunk 1 above)
Comment (Deepanshu0703) [reply components/GoToTop.js]: Removed animantion

**Review by anmode — APPROVED** [review]
Cool! Really liked the approach!

**Comment by AmanJain18** [comment]
can u reduced the size of the button a little bit for all screen sizes, it to big, and change background of button that matchs to oue website theme, you can show colour theme to us in discord dm before commiting

**Comment by AmanJain18** [comment]
@Deepanshu0703 Which issue u are working on let me know...

**Comment by Deepanshu0703** [comment]
#65

**Comment by AmanJain18** [comment]
> #65 

Left a comment above, please check it

**Comment by Deepanshu0703** [comment]
Ok I will decrease the size and change the color and will commit those changes in the morning . And ya I also want to know whether a lighter version of black is good or any other color . I will send some of the them while implementing on discord. Thanks

**Comment by AmanJain18** [comment]
bhai git pull karo ek baar

**Comment by Deepanshu0703** [comment]
mene bas apni repo sync kari koi error h kya

**Comment by AmanJain18** [comment]
local pai dekho apne home page ka form change hua kya design
and when u willl send pic of button, i wamt to merge this asap!

**Comment by Deepanshu0703** [comment]
image
 ye h

**Comment by AmanJain18** [comment]
commit new changes

**Comment by AmanJain18** [comment]
Finally merging Congratulations @Deepanshu0703 🎉🎉🚀💪🏻

**Comment by Deepanshu0703** [comment]
Thanks

**Pull request merged**